templates/blog/article.html

```diff
@@ -29,6 +29,12 @@
     </div>
   </section>
 
+  {% if empty %}
```

Comment:
If you use `errors` as a list of errors, we could even extract [the specific errors rendering template portion](https://github.com/neomad-team/neomad.org/blob/master/templates/auth/login.html#L65) to move it into an [includable/reusable template portion](http://jinja.pocoo.org/docs/2.9/templates/#include)